content/docs/tasks/telemetry/gateways/index.md

```diff
@@ -0,0 +1,632 @@
+
+{{< warning >}}
+This task is concerned *solely* with securing the transport layer for access. When exposing the telemetry
+services externally, the services should be installed and configured to require authentication.
+{{< /warning >}}
+
+1. [Install Istio](/docs/setup/) in your cluster with `cert-manager` enabled and `istio-ingressgateway` configured to use
+the [Secret Discovery Service](https://www.envoyproxy.io/docs/envoy/latest/configuration/secret#config-secret-discovery-service).
+
+    Use the following helm install options:
+
+    * `--set gateways.enabled=true`
+    * `--set gateways.istio-ingressgateway.enabled=true`
+    * `--set gateways.istio-ingressgateway.sds.enabled=true`
+    * `--set certmanager.enabled=true`
+    * `--set certmanager.email=mailbox@donotuseexample.com`
+
+    Install the telemetry add-ons using the following helm install options:
+
+    * Grafana: `--set grafana.enabled=true`
+    * Kiali: `--set kiali.enabled=true`
+    * Prometheus: `--set prometheus.enabled=true`
+    * Tracing: `--set tracing.enabled=true`
+
+1. Configure the DNS records for your domain.
+
+    1. Get the External IP Address of the `istio-ingressgateway`.
+
+        {{< text bash >}}
+        $ kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+        <IP ADDRESS OF CLUSTER INGRESS>
+        {{< /text >}}
+
+    1. Set an environment variable to hold your target domain.
+
+        {{< text bash >}}
+        $ TELEMETRY_DOMAIN=<your.desired.domain>
+        {{< /text >}}
+
+    1. Point your desired domain at that External IP Address via your domain provider.
+
+    1. Verify that the DNS records are correct.
+
+        {{< text bash >}}
+        $ dig +short $TELEMETRY_DOMAIN
+        <IP ADDRESS OF CLUSTER INGRESS>
+        {{< /text >}}
+
+1. Generate a server certificate
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: certmanager.k8s.io/v1alpha1
+    kind: Certificate
+    metadata:
+      name: telemetry-gw-cert
+      namespace: istio-system
+    spec:
+      secretName: telemetry-gw-cert
+      issuerRef:
+        name: letsencrypt
+        kind: ClusterIssuer
+      commonName: $TELEMETRY_DOMAIN
+      dnsNames:
+      - $TELEMETRY_DOMAIN
+      acme:
+        config:
+        - http01:
+            ingressClass: istio
+          domains:
+          - $TELEMETRY_DOMAIN
+    ---
+    EOF
+    certificate.certmanager.k8s.io "telemetry-gw-cert" created
+    {{< /text >}}
+
+1. Wait until the server certificate is ready.
+
+    {{< text bash >}}
+    $ JSONPATH='{range .items[*]}{@.metadata.name}:{range @.status.conditions[*]}{@.type}={@.status}{end}{end}' && kubectl -n istio-system get certificates -o jsonpath="$JSONPATH"
+    telemetry-gw-cert:Ready=True
+    {{< /text >}}
+
+1. Apply networking configuration for the telemetry add-ons.
+
+   1. Apply the Grafana configuration.
```

Comment:
Be specific about to what object/component you are applying the configuration to and with what purpose.

```suggestion
   1. Apply the following configuration to enable Grafana:
```

Reply:
done.